test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Date.Tests.ps1

```diff
@@ -97,6 +97,49 @@ Describe "Get-Date DRT Unit Tests" -Tags "CI" {
         Get-Date -Date $date -UFormat %V | Should -BeExactly $week
     }
 
+    It "using -uformat 'u' produces the correct output" -TestCases @(
+        @{date="1998-01-02"; dayOfWeek = "5"},
+        @{date="1998-01-03"; dayOfWeek = "6"},
+        @{date="2003-01-03"; dayOfWeek = "5"},
+        @{date="2004-01-02"; dayOfWeek = "5"},
```

Comment:
Please clarify how you select the test values (whole list)?

Reply:
I just used the values from the previous test for %V, which is related. I'll add a comment to that effect.